src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newDate = [];
+  const dateArray = date.split(fromFormat[3]);
+  const dateObject = {};
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    dateObject[fromFormat[i]] = dateArray[i];
+  }
+
+  if (dateObject.YYYY) {
+    dateObject.YY = dateObject.YYYY.slice(2);
+  }
+
+  if (dateObject.YY < 30) {
+    dateObject.YYYY = `20${dateObject.YY}`;
+  } else {
+    dateObject.YYYY = `19${dateObject.YY}`;
```

Comment:
Please create for magic numbers constants. 30, 19, 20, 3...

Reply:
Accidently clicked approve, so you don't need re-request again.
Just pay attention on comment in next tasks